swc-uxp-react-starter/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "swc-uxp-react-starter",
+  "id": "swc-uxp-react-starter",
+  "version": "1.0.0",
+  "main": "index.html",
+  "manifestVersion": 4,
```

Comment:
I think you should use version: 5
"featureFlags" is not valid property in manifest version 4 ...according to schema